src/Randomizer.Shared/Models/LogicConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel;
+
+namespace Randomizer.Shared
+{
+    public class LogicConfig
+    {
+        [DisplayName("Prevent Screw Attack Sock Locks"), Description("Requires the morph ball before using screw attacks in locations where you can get stuck"), Category("Logic")]
+        public bool PreventScrewAttackSoftLock { get; set; }
+
+        [DisplayName("Require Ten Power Bombs"), Description("Always require at least two power bomb upgrades for navigation in Super Metroid. Will require tracking multiple power bombs."), Category("Logic")]
```

Comment:
I'd change "Always require" to "You're expected to have". Since this would effectively give you another set of power bombs early, maybe change the name to "More power bombs" or something? "Require" doesn't quite feel like the right word to me.